### ISSUE TITLE
Clean up rules 942180 and 942340

### ISSUE THREAD

**Issue by theseion** [coreruleset/coreruleset#2471]
As discussed in #2355, rules 942180 and 942340 both include tests for stuff that's not SQL injection related. These things should be covered by other rules. Move tests to other rules and remove the regex statements that match other things than SQL injection.

**Comment by dune73** [comment]
There is a lot of discussion around this and I see some need to move these tests.

I am not entirely sure I get which tests. Could you indentify them please @theseion.

Maybe it would also be OK to just drop them. A call we can make when we have defined which ones we are talking about.

**Comment by theseion** [comment]
Yes, I can do that. But I'm currently working on other issues that have a higher priority, so it might be a while.

**Comment by dune73** [comment]
OK, so let's leave this for now.

**Comment by Xhoenix** [comment]
There is a XSS payload in 942180 tests and a payload with "except selecting" in 942340 which is(selecting?) not in `942340.ra`?  The numbers for SQL authentication bypass also needs to be updated. Anything else I'm missing @theseion? If you're busy I can take this, you can later review it. :)

**Comment by Xhoenix** [comment]
I'm just trying to clear all old issues that are pending, you can check some of the oldest issues in this repo and you would find my comments. From the way it sounds, I'm not going to take this issue. Sorry. :(

**Comment by theseion** [comment]
np

**Comment by fzipi** [comment]
The "except selecting" in 942340 is a *negative* test. It was created for not matching the rule.

**Comment by fzipi** [comment]
Can we close this one?
